src/tools.rs

```diff
@@ -152,6 +152,7 @@ impl Application {
             Application::Sass => text
                 .lines()
                 .next()
+                .and_then(|s| s.split(' ').next())// allow for dart2js compiled version
```

Comment:
Can you add an example of how to use this? Maybe just a comment to the Trunk.toml example in this repo, or in one of the examples.

Reply:
As you can see in the test below the sass compiler outputs the dart2js version, if it's compiled with it. This code just strips everything, behind the sass compiler version, as it's the only thing we care about.

From a user perspective nothing changes, the value you add in the Trunk.toml is the same as with the native dart version of the sass compile. Should i expand the comment on this line, explaining why we're doing this?



```suggestion
                // If the sass compiler was build with `dart2js` it will append
                // the `dart2js` version number to it's own version output.
                // This version output looks like this: "1.37.5 compiled with dart2js 2.14.4"
                .and_then(|s| s.split(' ').next())
```

Reply:
I think this is alright. You added a test case as well, to make sure it works properly.

@thedodd it seems to be popular to use `dart2js` to transliterate the original `dart-sass` into JS, to make it easier to deploy/run on systems that don't support dart natively. Plus, you don't have to ship the whole dart runtime with it.

On macOS with homebrew it's similar. The dart-sass maintainers have their own "tap" (like a separate repo from the main one) with an install script for dart-sass, depending on dart. But the main/core repo contains only `node-sass` which is dart-sass converted with `dart2js`.

As described by @xanderio, programs converted with dart2js don't just contain the original version of the program, but the dart2js converter version as well. That is in the form of `<program-version> compiled with dart2js <dart2js-version>`, and we currently don't extract the version properly.

This PR fixes it so we get only the `<program-version>` part, and not the whole string (which we don't want/need).